dotnet/src/SemanticKernel/Planning/Plan.cs

```diff
@@ -255,9 +252,7 @@ public async Task<Plan> InvokeNextStepAsync(SKContext context, CancellationToken
             var functionVariables = this.GetNextStepVariables(context.Variables, step);
```

Comment:
If we pass in `variables` this will become

`var functionVariables = this.GetNextStepVariables(variables, step);`

Reply:
Good suggestion. 
All plan specific structural/logic changes will be in a next phase.